registration/tests/views/test_field.py

```diff
@@ -51,12 +51,23 @@ def test_get_fields(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(payload, serializer.data)
 
-    def test_method_not_allowed(self):
+    def test_post_fields(self):
         url = reverse("field-list")
         self.client.force_authenticate(self.user)
-
-        response = self.client.post(url)
-        self.assertEqual(response.status_code, status.HTTP_405_METHOD_NOT_ALLOWED)
+        response = self.client.post(
+            url,
+            {
+                "role": "Child",
+                "name": "Favourite Colour",
+                "question": "What is your favourite colour",
+                "question_type": "Text",
+                "is_default": False,
+                "options": [],
+                "order": Field.objects.filter(role="Child").count() + 1,
```

Comment:
are the # of fields and the order expected to be equal? I think with the current validator this isn't necessarily the case, so it would be clearer to reflect that logic here!

Reply:
fixed the validator instead :D